### PR TITLE
zh-cn: translate for Credential API

### PR DESCRIPTION
**Pull request by xuanzhi33**
### Description

zh-CN: translate for Credential API

### CODE REVIEW

**Comment by xuanzhi33** [comment]
Sorry, I accidentally added a space in the file path

File: files/zh-cn/web/api/credential/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- {{domxref("PasswordCredential")}}
+- {{domxref("PublicKeyCredential")}}
+- {{domxref("OTPCredential")}}
+
+## 实例属性
+
+- {{domxref("Credential.id")}} {{ReadOnlyInline}}
+  - : 返回包含凭据标识符的字符串。这可以是 GUID、用户名或电子邮件地址。
+- {{domxref("Credential.type")}} {{ReadOnlyInline}}
+  - : 返回包含凭据类型的字符。有效值为 `password`、`federated`、`public-key`、`identity` 和 `otp`. (分别对应 {{domxref("PasswordCredential")}}、{{domxref("FederatedCredential")}}、{{domxref("PublicKeyCredential")}}、{{domxref("IdentityCredential")}} 和 {{domxref("OTPCredential")}})
```
Comment (yin1999) [inline files/zh-cn/web/api/credential/index.md:23]:
```suggestion
  - : 返回包含凭据类型的字符。有效值为 `password`、`federated`、`public-key`、`identity` 和 `otp`。（分别对应 {{domxref("PasswordCredential")}}、{{domxref("FederatedCredential")}}、{{domxref("PublicKeyCredential")}}、{{domxref("IdentityCredential")}} 和 {{domxref("OTPCredential")}}）
```

File: files/zh-cn/web/api/credential/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: Credential
+slug: Web/API/Credential
+---
```
Comment (yin1999) [inline files/zh-cn/web/api/credential/index.md:4]: Let's add the sourceCommit [metadata](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)

```suggestion
l10n:
  sourceCommit: 952db471d65c30302c44388edf3c0aa094874319
---
```

**Review by yin1999 — APPROVED** [review]
Thank you @xuanzhi33. Welcome to the community :tada:

**Pull request merged**